crates/cxx-qt-build/Cargo.toml

```diff
@@ -13,7 +13,7 @@ description = "Helpers for integrating `cxx-qt` crate into a Cargo build"
 repository.workspace = true
 
 [dependencies]
-cc = "1.0.74"
+cc = { version = "1.0.74", features = ["parallel"] }
```

Comment:
We had this before, then we reverted it or it was lost ? #220 was there a reason ? eg Windows file locks or something ?

Reply:
Oh no that's just `cxx-build` :-) So maybe we just missed enabling it for our own `cc` crate